nbclassic/notebookapp.py

```diff
@@ -126,6 +127,7 @@ class NotebookApp(
     aliases = aliases
     flags = flags
     extension_url = "/tree"
+    subcommands = {}
```

Comment:
This should fix the issue mentioned in https://github.com/jupyterlab/nbclassic/pull/45#issuecomment-784233906.

Although it's possible an upstream fix `jupyter_server` could also work.

Reply:
:+1: